test/features/login.feature

```diff
@@ -1,17 +1,6 @@
 @smoke @login
 Feature: Login
 
-    Scenario: Smoke test to ensure Login process works
```

Comment:
why was this removed?

Reply:
It's redundant. It's manually performing the same actions as the "I log in" step from the other test.